HIPs/hip-1.md

```diff
@@ -0,0 +1,271 @@
+
+It is highly recommended that a single HIP contain a single key proposal or new idea. The more focused the HIP, the more successful it tends to be. A change to one client doesn't require an HIP; a change that affects multiple clients, or defines a standard for multiple apps to use, does.
+
+An HIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+
+### Special Requirements for Core HIPs
+
+If a **Core** HIP mentions or proposes changes to the Covenent systems, it should refer to the instructions by their mnemonics and define the opcodes of those mnemonics at least once. A preferred way is the following:
+```
+TRANSFER (0xfe)
+```
+
+## HIP Work Flow
+
+### Shepherding an HIP
+
+Parties involved in the process are you, the champion or *HIP author*, the [*HIP editors*](#hip-editors), and the [*handshake Client Developers*](#client-developers).
+
+Before you begin writing a formal HIP, you should vet your idea. Ask the Handshake community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is thus recommended to open a discussion thread on [the Handshake Magicians forum] to do this, but you can also use [one of the Handshake Gitter chat rooms], [the Handshake subreddit] or [the Issues section of this repository]. 
+
+In addition to making sure your idea is original, it will be your role as the author to make your idea clear to reviewers and interested parties, as well as inviting editors, developers and community to give feedback on the aforementioned channels. You should try and gauge whether the interest in your HIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the work required for implementing a Core HIP will be much greater than for an ERC and the HIP will need sufficient interest from the Handshake client teams. Negative community feedback will be taken into consideration and may prevent your HIP from moving past the Draft stage.
+
+### Core HIPs
+
+For Core HIPs, given that they require client implementations to be considered **Final** (see "HIPs Process" below), you will need to either provide an implementation for clients or convince clients to implement your HIP. 
+
+The best way to get client implementers to review your HIP is to present it on an AllCoreDevs call. You can request to do so by posting a comment linking your HIP on an [AllCoreDevs agenda GitHub Issue].  
```

Comment:
Again, no AllCoreDevs call. Lets just try to keep everything in this github repo for now. Maybe also mention #handshake on IRC? I'm wary of even mentioning telegram channels because they may be ephemeral.